helm/docs-app/templates/psp-rbac.yaml

```diff
@@ -0,0 +1,60 @@
+---
+apiVersion: extensions/v1beta1
+kind: PodSecurityPolicy
+metadata:
+    name: {{ .Values.name }}
+spec:
+    allowPrivilegeEscalation: false
+    fsGroup:
+    ranges:
+    - max: 65535
+        min: 1
+    rule: MustRunAs
+    runAsUser:
+    rule: MustRunAsNonRoot
+    seLinux:
+    rule: RunAsAny
+    supplementalGroups:
+    ranges:
+    - max: 65535
+        min: 1
+    rule: MustRunAs
+    volumes:
+    - secret
+    - configMap
+    - emptyDir
```

Comment:
the indentation on psp s broken , check this for comparsion https://github.com/giantswarm/happa/blob/master/helm/happa-chart/templates/psp.yaml

Reply:
Thanks! Needed to change my editor's indentation settings from the default 4 spaces...